### PR TITLE
Payment overview update

### PR DESCRIPTION
**Pull request by ItsANameToo**
I've added the pending balance and the total amount received to the payment history screen (mostly so I don't have to go through that long list if I want to see my currently pending balance :P)

Currently it uses the `https://pool.arkcoin.net/voters/rewards` endpoint to retrieve the pending balance, but this also returns the pending balances for all the other voters. Is there any way to retrieve the value for just a single address? I went over the `arkgoserver` documentation, but couldn't find any info on that.

Also small change to the readme file, as those `npm install` commands were redundant (everything is already installed after the initial `npm install` command).

Edit: screenshot
balances

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,6 @@ Dashboard for arkcoin.net
 git clone https://github.com/Jarunik/arkcoin
 cd arkcoin
 npm install
```
Comment (Jarunik) [inline README.md]: I like to have the additional modules for documentation purposes.

File: README.md [hunk 2]
```diff
@@ -22,6 +17,8 @@ npm run
 npm start
 ```
```
Comment (Jarunik) [inline README.md]: Not sure why that comment is needed. It is starting up automatically anyway.

File: README.md (hunk 1 above)
Comment (ItsANameToo) [reply README.md]: Line of thought was that they are also documented in the package.json, so therefore I found it unneccessary to list them here too

File: README.md (hunk 2 above)
Comment (ItsANameToo) [reply README.md]: Good point

**Comment by Jarunik** [comment]
I don't think arkgo has a better API for it (yet). At least rewards are cached now so it is reasonable fast after the first access.

**Pull request merged**